src/absences/absences.test.ts

```diff
@@ -37,7 +40,52 @@ describe('Absences', () => {
 
   test('readById', async () => {
     mock.onGet(`${readPath}/1`).reply(200, { Success: true, NumResults: 1, Results: [{}] });
-    resultSingle = absences.readById(1);
-    await resultSingle.then((result) => expect(result).toStrictEqual({}));
+    const result: Absence = await absences.readById(1);
+    expect(result).toStrictEqual({});
+  });
+
+  test('create', async () => {
+    const absence = { user_id: 1, from_date: '01-01-2020', to_date: '03-01-2020', type_id: 2, subtype_id: 3 };
```

Comment:
Is there any reason that we need to have "real" data here? Especially when the data doesn't make any sense.

Reply:
Not necessarily need, but I believe it is better than using just `{}` in the tests, where we cannot detect at all if there is some incorrect change or mixing up of fields somewhere between receiving the response and returning from eg. `timeTrackings.read()`. Not necessary at the moment since there is no logic there, but could become relevant if some sort of transformation needs to be done. With "doesn't make any sense", were you referring to the previously incorrect date format?